src/modules/formBuilder/components/FormBuilderPalette.tsx

```diff
@@ -119,22 +119,21 @@ interface FormBuilderPaletteType {
 const FormBuilderPalette: React.FC<FormBuilderPaletteType> = ({
   onItemClick,
 }) => {
+  const top = STICKY_BAR_HEIGHT + CONTEXT_HEADER_HEIGHT;
+
   return (
-    <Drawer
-      variant='persistent'
-      open={true}
+    <Paper
+      square
       sx={{
-        flexShrink: 0,
-        '& .MuiDrawer-paper': {
-          height: `calc(100vh - ${
-            STICKY_BAR_HEIGHT + CONTEXT_HEADER_HEIGHT
-          }px)`,
-          position: 'static',
-          borderTop: 'none',
-          boxSizing: 'border-box',
-          p: 1,
-          pt: 2,
-        },
+        height: `calc(100vh - ${top}px)`,
+        position: 'sticky',
+        left: 'auto',
```

Comment:
For some reason setting `right: 0` leaves the palette hanging out in the middle of the screen when the left-hand admin dashboard is toggled closed. `left: auto` seemed to do the trick though, I haven't observed any similar bugs with it.